guacamole-docker/README.md

```diff
@@ -228,6 +228,49 @@ The process for doing this via the `sqlcmd` utilities included
 with SQLServer is documented in
 [the Guacamole manual](http://guacamole.apache.org/doc/gug/jdbc-auth.html#jdbc-auth-sqlserver).
 
+Enabling guacd ssl
+================
+This explains how to enable ssl between guacamole and guacd using a self signed certificat.
+
+1. Generate a new certificat
+You need to create the new certificat on the guacd host.
+
+```shell
+openssl genrsa -out /etc/guacd/server.key 2048
+openssl req -new -key /etc/guacd/server.key -out /etc/guacd/cert.csr
+openssl x509 -in /etc/guacd/cert.csr -out /etc/guacd/server.crt -req -signkey /etc/guacd/server.key -days 3650
+openssl pkcs12 -export -in /etc/guacd/server.crt -inkey /etc/guacd/server.key  -out /etc/guacd/server.p12 -CAfile ca.crt -caname root
+```
+2. Configure guacd
+
+On debian, edit /etc/default/guacd and modify the following variables.
+```
+# listen on all interface
+LISTEN_ADDRESS=0.0.0.0
+
+# certificats
+DAEMON_ARGS=-C /etc/guacd/server.crt -K /etc/guacd/server.key
+```
+restart guacd! 
```

Comment:
Why are these steps required for a Docker install? Wouldn't this be steps that would work if you're running guacd natively?

Reply:
@necouchman ,
Yes, I added documentation on how to setup the guacd standalone only with ssl mode.

The guacd doc should probably go somewhere else indeed. I don't think it can go in the Docker chapter in the User Guide since one can use either the guacd standalone or guacd docker or both.

I don't know where it should go ? What would you recommend ?

Reply:
This may be more suited for the Configuring guacd section of the user guide.

Reply:
@necouchman 
can I use the same Jira to update guacd section of the user guide ?

Reply:
Sure, I think that would be fine.

Reply:
All is now over here
[guacamole-server/pull/419](https://github.com/apache/guacamole-server/pull/419)